.github/ISSUE_TEMPLATE/new-package-request.md

```diff
@@ -1,35 +1,35 @@
 ---
 name: New Package Request
-about: Suggest inclusion of a new package / application in the Flatcar OS image
+about: Propose inclusion of a new package into Flatcar Container Linux
 title: 'New Package Request: [package-name]'
 labels: "kind/new-package"
 assignees: ''
 
 ---
 
-**Package name and purpose**
+**Package name and purpose**  
 [ Please briefly describe the new package. The description should be generic; please use the other sections below to elaborate on its role for Flatcar ]
 
 - Package upstream repo: [ link to the package’s upstream repo ]
 - Gentoo ebuild: [ link to package’s Gentoo ebuild if exists ]
 
-**Impact of adding this package to the Flatcar OS image**
-- [ ] This package meets the requirements of Flatcar Container Linux’ new package guidelines: https://github.com/flatcar-linux/Flatcar/blob/master/adding-new-packages.md
+**Impact of adding this package to the Flatcar OS image**  
```

Comment:
IMO, we should be consistent with the Flatcar naming, so we should be using Flatcar Container Linux or FCL everywhere.